src/Plivo/Resources/Message/MessageInterface.php

```diff
@@ -76,45 +64,45 @@ public function get($messageUuid)
      */
     protected function getList($optionalArgs = [])
     {
-        $response = $this->client->fetch(
-            $this->uri,
-            $optionalArgs
-        );
+        $response = $this
+            ->client
+            ->fetch($this->uri, $optionalArgs);
 
         $messages = [];
 
-        foreach ($response->getContent()['objects'] as $message) {
+        foreach ($response->getContent() ['objects'] as $message)
+        {
             $newMessage = new Message($this->client, $message, $this->pathParams['authId'], $this->uri);
 
             array_push($messages, $newMessage);
         }
 
-        return new MessageList($this->client, $response->getContent()['meta'], $messages);
+        return new MessageList($this->client, $response->getContent() ['meta'], $messages);
```

Comment:
do we need a space here ?$response->getContent() ['meta']

Reply:
Fixed